src/application/components/Application.js

```diff
@@ -59,6 +59,7 @@ export class Application extends Component {
       } = this.props.params;
 
       const operation = utils.getOperation(resource, method, path, service);
+      const resourceObj = utils.getResource(resource, service);
```

Comment:
Call this resource. eg
const resource = utils.getResource(this.props.params.resource, service)
Remove resource from the destructure